changelog/v1.19.0-rc4/lambda-persist-request-details.yaml

```diff
@@ -0,0 +1,10 @@
+changelog:
```

Comment:
the changelog should be in folder `changelog/v1.19.0-patch4/lambda-persist-request-details.yaml`